fMRI_Tutorial/DataCheck.ipynb

```diff
@@ -0,0 +1,473 @@
+      "text/plain": [
+       "<Figure size 1080x1080 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Visualize entire matrix in its raw form\n",
+    "plt.figure(figsize=(15,15))\n",
+    "sns.heatmap(matfinal, cmap='coolwarm')"
```

Comment:
Is coolwarm good for colorblind folks? Thought a unicolor scale with increasing/decreasing saturation worked best for all types of colorblindness?

Reply:
I thought coolwarm red/blues were ok according to some tutorials, but I've changed to a colorblind palette from seaborn. It's not the most beautiful thing, but should be more inclusive this way :)